### ISSUE TITLE
page animation transition

### ISSUE THREAD

**Issue by glendonC** [herohubUVA/herohub2.0#1]
I had a lot of problems trying to integrate this feature (specifically based on this template: https://codepen.io/alvarotrigo/pen/wvpbRmE) so for future reference if we try to implement it, this was my main issue:

getting the animation to begin to cover the screen was fine. Still, when the screen was covered and I had a loading SVG icon, I tried to make it so that the newly routed page would load such that when the reverse animation started it looked like the new page was being unveiled.

in the beginning, I had the whole animation work (left to right tiles, SVG, right to left), but it wouldn't preload the page I just did the animation and then loaded the page, which is redundant.

Then, I figured that I would have to integrate AJAX to preload the page as the animation covered the old page, but then the CSS/JS styles weren't rendering on the new pages and it was a mess trying to configure the routes alongside that i.e. one of the problems was that I would go from Home @ http://localhost:4000/ to Character Encyclopedia, but because of AJAX, it would stay as http://localhost:4000/ but load the search container content that was unique to that page. The only problem was that as mentioned before, the CSS and JS weren't rendering and then it got very convoluted when I figured out that I could manually search for http://localhost:4000/CharacterEncyclopedia and that link had the working JS/CSS, but I couldn't route to it in the new AJAX configuration.

tldr; ajax was very confusing and page transition animation will be standby
